deployments/forgeops/README.md

```diff
@@ -36,6 +36,77 @@ DS Password: zMO2W9IlOronDqrF2MtEha3Jiic3urZM
 =====================================================
 ```
 
+### Example
```

Comment:
```suggestion
### Authenticate and Authorize a Thing
```

Reply:
I'm not sure about this new title since the authentication bit in the example is very simplistic. The focus is on the `things` endpoint so how about "Using the `things` endpoint"?

Reply:
It feels a bit strange to go directly to the `things` endpoint after starting the platform. Could we add a paragraph at the start describing what we're doing. How about this:

### Using the Platform for Things

Once the Platform is running we can register and authenticate a thing via the `authenticate` endpoint and perform actions like authorization and retrieving attributes via the `things` endpoint. This can either be done with the IoT SDK or by using the endpoints directly.

The `things` endpoint...